src/app/components/SearchHeader/components/SearchExtended/styles.js

```diff
@@ -0,0 +1,22 @@
+import { StyleSheet } from 'react-native';
+import colors from '@constants/colors';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    backgroundColor: colors.white,
+    alignItems: 'center',
+    height: 40,
+    borderRadius: 20
+  },
+  input: {
+    width: 300
+  },
+  icon: {
+    width: 15,
+    height: 15,
```

Comment:
extract common constant value

Reply:
Done! (I don't like me that approach but, i changed it anyway)